### PR TITLE
feat(STONEBLD-1773): alerting for SLOs for build-service

### PR DESCRIPTION
**Pull request by rcerven**

### CODE REVIEW

**Comment by amisstea** [comment]
@rcerven please include unit tests for these alerting rules. When you do, please also split this PR by rule to make the review process go quicker.

File: rhobs/alerting/data_plane/prometheus.latency_component_onboarding_alerts.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        severity: warning
+        slo: "true"
+      annotations:
+        summary: >-
+          Latency of component onboarding time exceeded
+        description: >
+          Time spent from the moment of Component creation till the initial simple build is submitted,
+          if PaC provision is requested upon the Component creation, then till the provision finishes has been over
+          60s for more than 10% of requests during the last 5 minutes on cluster
+          {{ $labels.source_cluster }}
```
Comment (amisstea) [inline rhobs/alerting/data_plane/prometheus.latency_component_onboarding_alerts.yaml:33]: Please include the actual expression value in each description.
```suggestion
          Time spent from the moment of Component creation till the initial simple build is submitted,
          if PaC provision is requested upon the Component creation, then till the provision finishes has been over
          60s for {{ value | humanizePercentage }} of requests during the last 5 minutes on cluster
          {{ $labels.source_cluster }}
```

File: rhobs/alerting/data_plane/prometheus.latency_pac_unprovision_alerts.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      labels:
+        severity: warning
+        slo: "true"
+      annotations:
+        summary: >-
+          Latency of component PaC unprovision time exceeded
+        description: >
+          Time taken from PaC unprovision request till Component is unprovisioned for PaC builds has been over
+          20s for more than 5% of requests during the last 5 minutes on cluster
+          {{ $labels.source_cluster }}
```
Comment (amisstea) [inline rhobs/alerting/data_plane/prometheus.latency_pac_unprovision_alerts.yaml:32]:
```suggestion
          Time taken from PaC unprovision request till Component is unprovisioned for PaC builds has been over
          20s for {{ value | humanizePercentage }} of requests during the last 5 minutes on cluster
          {{ $labels.source_cluster }}
```

File: rhobs/alerting/data_plane/prometheus.latency_simple_build_alerts.yaml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+      labels:
+        severity: warning
+        slo: "true"
+      annotations:
+        summary: >-
+          Latency of component simple build submission time exceeded
+        description: >
+          Time taken from simple build request till the build pipeline is submitted has been over
+          15s for more than 5% of requests during the last 5 minutes on cluster
+          {{ $labels.source_cluster }}
```
Comment (amisstea) [inline rhobs/alerting/data_plane/prometheus.latency_simple_build_alerts.yaml:32]:
```suggestion
          Time taken from simple build request till the build pipeline is submitted has been over
          15s for {{ value | humanizePercentage }} of requests during the last 5 minutes on cluster
          {{ $labels.source_cluster }}
```

File: rhobs/alerting/data_plane/prometheus.stability_pac_provision_alerts.yaml [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+      labels:
+        severity: warning
+        slo: "true"
+      annotations:
+        summary: >-
+          Stability of component PaC provision time exceeded
+        description: >
+          Time taken from PaC provision request till Component is provisioned for PaC builds has been over
+          5 minutes for more than 1% of requests during the last 10 minutes on cluster
+          {{ $labels.source_cluster }}
```
Comment (amisstea) [inline rhobs/alerting/data_plane/prometheus.stability_pac_provision_alerts.yaml:32]:
```suggestion
          Time taken from PaC provision request till Component is provisioned for PaC builds has been over
          5 minutes for {{ value | humanizePercentage }} of requests during the last 10 minutes on cluster
          {{ $labels.source_cluster }}
```

File: rhobs/alerting/data_plane/prometheus.stability_pac_unprovision_alerts.yaml [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+      labels:
+        severity: warning
+        slo: "true"
+      annotations:
+        summary: >-
+          Stability of component PaC unprovision time exceeded
+        description: >
+          Time taken from PaC unprovision request till Component is unprovisioned for PaC builds has been over
+          5 minutes for more than 1% of requests during the last 10 minutes on cluster
+          {{ $labels.source_cluster }}
```
Comment (amisstea) [inline rhobs/alerting/data_plane/prometheus.stability_pac_unprovision_alerts.yaml:32]:
```suggestion
          Time taken from PaC unprovision request till Component is unprovisioned for PaC builds has been over
          5 minutes for {{ value | humanizePercentage }} of requests during the last 10 minutes on cluster
          {{ $labels.source_cluster }}
```

File: rhobs/alerting/data_plane/prometheus.stability_simple_build_alerts.yaml [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+      labels:
+        severity: warning
+        slo: "true"
+      annotations:
+        summary: >-
+          Stability of component simple build submission time exceeded
+        description: >
+          Time taken from simple build request till the build pipeline is submitted has been over
+          5 minutes for more than 1% of requests during the last 10 minutes on cluster
+          {{ $labels.source_cluster }}
```
Comment (amisstea) [inline rhobs/alerting/data_plane/prometheus.stability_simple_build_alerts.yaml:32]:
```suggestion
          Time taken from simple build request till the build pipeline is submitted has been over
          5 minutes for {{ value | humanizePercentage }} of requests during the last 10 minutes on cluster
          {{ $labels.source_cluster }}
```

**Comment by rcerven** [comment]
we don't need to report specific percentage because if alert goes above threshold it is already issue for us

**Comment by amisstea** [comment]
@rcerven please rebase this PR when you get a chance. The check failures are probably related to recent updates in #147 . Attn: @hmariset

**Comment by rcerven** [comment]
rebased

**Review by amisstea — APPROVED** [review]
Will merge even though the new on-pull-request pipelinerun isn't being triggered (due to recent changes in #147). The older version verified the important parts anyway.

**Pull request merged**